README.md

```diff
@@ -1,2 +1,14 @@
 # CAN-Explorer
 An application to visualize CAN data and interact with the CAN bus
+
+Clone the repository :
+git clone "url"
+
+Make a branch :
+git checkout -b <branch_name>
+
+Install node modules :
+npm install
```

Comment:
I don't think our VM has npm installed by default (at least I don't). Consider adding instructions at least a link to something like this here: https://github.com/nodesource/distributions

Reply:
I'm also not sure, what verison you're using? Probably best to write that down too.

Reply:
In my case, I had to run
```
curl -sL https://deb.nodesource.com/setup_15.x | sudo -E bash -
sudo apt-get install -y nodejs
```